cartridges/rvw_dev_console/cartridge/scripts/serializer.js

```diff
@@ -99,6 +99,11 @@ function serializeObject (object, maxDepth, depth, pojo) {
             continue;
         }
 
+        // Form Instances seem to create endless loops, so let's ignore them
+        if (object instanceof dw.web.Form) {
```

Comment:
This cant be. The ability to show forms is a very important piece to this and we aren't stripping that out. There is something wrong on your sandbox or in the code, but this is not the answer.

I use this to check out form data on a daily basis for years now.

Reply:
I will remove that, I forgot that the two sites I was testing this on were actually based on the same site import.